### PR TITLE
Added bitwuzla to pysmt frontend

### PR DESCRIPTION
**Pull request by jack-melchert**
Added bitwuzla to pysmt frontend, existing python tests are sufficient to test this.

### CODE REVIEW

File: python/smt_switch/pysmt_frontend/pysmt_solver.py [hunk 1]
```diff
@@ -199,6 +199,22 @@ def _reset_assertions(self):
     SWITCH_SOLVERS['btor'] = SwitchBtor
 
 
+if 'bitwuzla' in  ss.solvers:
+    logics_params = dict(
+        quantifier_free=[True],
+        arrays=[True, False],
```
Comment (yoni206) [inline python/smt_switch/pysmt_frontend/pysmt_solver.py:205]: What does `[True, False]` mean in this context? Same for `uninterpreted` and `floating_point` below?

Comment (jack-melchert) [reply python/smt_switch/pysmt_frontend/pysmt_solver.py:205]: Basically this is just an easy way of generating all of the supported logics for the solver. The list for each `logics_params `entry is the available settings for that option, with the default being [False]. Every possible combination of these options is then used to generate the list of supported logics.
For example, `quantifier_free=[True], bit_vectors=[True], uninterpreted=[True, False]` would result in supported logics of `QF_BV, QF_UFBV`.

Comment (yoni206) [reply python/smt_switch/pysmt_frontend/pysmt_solver.py:205]: So what does the `False` mean in the `[True,False]` part of, say, `uninterpreted`?

Comment (jack-melchert) [reply python/smt_switch/pysmt_frontend/pysmt_solver.py:205]: We want to generate every supported logic for this solver, which in this case includes both QF_BV and QE_UFBV, the `[True,False]` for uninterpreted says that the solver supports both. We are generating the list of supported logics from the every possible combination of these True/False lists.

**Review by yoni206 — APPROVED** [review]

**Comment by jack-melchert** [comment]
@yoni206 would you be able to merge this? I don't have write access to smt-switch.

**Pull request merged**